src/index.html

```diff
@@ -1,14 +1,34 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="first block" id = "qa-block">
+      <div class="second block">
+        <div class="third block">
+          In programming languages, a closure is a technique for implementing lexically<br>scoped name binding in a language with<br>first-class functions. Operationally, a<br>closure is a record storing a function<br>together with an environment.
+        </div>
+      </div>
+    </div>
+    <div class="first middle block">
+      <div class="second block">
+        <div class="third block">
```

Comment:
what if you have 100 blocks and then you decide to remove the 10th, will you rename all others to match numbers?

use logical names, e.g. outer, middle, inner